lib/std/math/big/int.zig

```diff
@@ -534,7 +534,8 @@ pub const Int = struct {
         return out_stream.writeAll(str);
     }
 
-    /// Returns -1, 0, 1 if |a| < |b|, |a| == |b| or |a| > |b| respectively.
+    /// Returns math.Order.lt, math.Oder.eq, math.Order.gt if |a| < |b|, |a| ==
```

Comment:
Typo, `Oder` -> `Order`

Reply:
Oops, will fix that in all of the doc comments